src/transformers/models/nystromformer/modeling_nystromformer.py

```diff
@@ -43,7 +43,7 @@
 
 _CHECKPOINT_FOR_DOC = "uw-madison/nystromformer-512"
 _CONFIG_FOR_DOC = "NystromformerConfig"
-_TOKENIZER_FOR_DOC = "AutoTokenizer"
+_TOKENIZER_FOR_DOC = "AlbertTokenizer"
```

Comment:
Disagree with that change here, we actually want people using the Auto APIs (so that they don't have to remember that Nystromformer should be used with AlbertTokenizer for instance). All model examples should eventually be switched to using the auto classes, so this is a step in the wrong direction.

Reply:
I am OK to change it back here. I understand that using Auto APIs is much easier.

I was just thinking that people might (sometimes) want to know which tokenizer is actually used, and thought putting the exact type is good.

Reply:
It's in the documentation page of the model already I believe.